### DISCUSSION TITLE
fly_to vs camera_position

### DISCUSSION THREAD

**Discussion by ENIAC-6** [pyvista/pyvista#3007] [Q&A]
In trying to make an example for fly_to, I was looking at modifying some code from the tutorial that uses camera_position and two questions came up. 1) I am curious, am I understanding correctly that fly_to is a method, and camera_position a property (I see it's listed under attributes in issue #1629. 

Also, my example turned out looking odd and was wondering if you had any suggestions:
```python
pl = pyvista.Plotter()
_ = pl.add_volume(
    dataset,
    cmap="coolwarm",
    opacity="sigmoid_5",
    show_scalar_bar=False,
)
#pl.camera_position = [(194.6, -141.8, 182.0), (34.5, 61.0, 32.5), (-0.229, 0.45, 0.86)]
pl.fly_to((1,1,1))
pl.show()
```

**Comment by akaszynski** [comment]
Good question, and looking at the doc string I think we’re going to have to clean it up a bit.

[fly_to](https://docs.pyvista.org/api/plotting/_autosummary/pyvista.Plotter.fly_to.html) animates as you move towards a point. I think it’s more useful when using the [BackgroundPlotter](https://qtdocs.pyvista.org/api_reference.html)
